response_operations_ui/error_handlers.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from requests import RequestException
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
```

Comment:
We probably want a `@app.errorhandler(Exception)` here too?

Reply:
We'll probably add this in when we get our generic error pages from RAD